dev/deploy_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import cortex
+import sys
+
+cx = cortex.client(sys.argv[1])
+api_config = {
+    "name": "text-generator",
+    "kind": "RealtimeAPI",
+}
+
+
+class PythonPredictor:
+    def __init__(self, config):
+        from transformers import pipeline
+
+        self.model = pipeline(task="text-generation")
+
+    def predict(self, payload):
+        return self.model(payload["text"])[0]
+
+
+api = cx.deploy(
+    api_config,
+    predictor=PythonPredictor,
+    requirements=["torch", "transformers"],
+    wait=True,
+)
+
+import requests
```

Comment:
mostly out of curiosity: is there a reason to import here instead of at the top?